run-checks

```diff
@@ -210,7 +210,7 @@ if [ "$STATIC" = 1 ]; then
 
     echo "Checking spelling errors"
     if ! command -v misspell >/dev/null; then
-        go get -u github.com/client9/misspell/cmd/misspell
+        go install github.com/client9/misspell/cmd/misspell@latest
```

Comment:
Running this locally (with 1.13) fails due to the `@latest` suffix. It works with some later versions like go1.17